Source/WebKit/Shared/WebProcessCreationParameters.cpp

```diff
@@ -496,6 +497,12 @@ bool WebProcessCreationParameters::decode(IPC::Decoder& decoder, WebProcessCreat
     if (!systemHasAC)
         return false;
     parameters.systemHasAC = WTFMove(*systemHasAC);
+    
+    std::optional<bool> strictSecureDecodingForAllObjCEnabled;
+    decoder >> strictSecureDecodingForAllObjCEnabled;
+    if (!strictSecureDecodingForAllObjCEnabled)
+        return false;
+    parameters.strictSecureDecodingForAllObjCEnabled = WTFMove(*strictSecureDecodingForAllObjCEnabled);
```

Comment:
No need for WTFMove()

Reply:
true.  Doesn't hurt either.

Reply:
The same way that unnecessary casts hurt. They hurt readability for no reason.